src/plugins/vis_types/timeseries/public/application/components/aggs/top_hit.js

```diff
@@ -204,7 +204,7 @@ const TopHitAggUi = (props) => {
               Should it be text or number?
             */}
             <input
```

Comment:
because previously we just have class which includes styles from `EuiFieldNumber`. I removed that class and add `EuiFieldNumber` class directlly for input. Mirgation of `input` to `euiFieldNumber`  is not a part of that task.

Reply:
Cool, was just curious if there was a specific technical reason not to use the `EuiFieldNumber` component. But if it goes beyond the scope of this PR, then no worries.

Reply:
I will create issue to migrate `input` to `EuiFieldNumber`